app/models/service_template.rb

```diff
@@ -234,12 +234,20 @@ def set_ownership(service, user)
     service.save
   end
 
-  def self.default_provisioning_entry_point
-    '/Service/Provisioning/StateMachines/ServiceProvision_Template/default'
+  def self.default_provisioning_entry_point(service_type = 'atomic', _prov_type = nil)
+    if service_type == 'atomic'
+      '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogItemInitialization'
+    elsif service_type == 'composite'
+      '/Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogBundleInitialization'
+    end
+  end
```

Comment:
@bzwei Also wanted to say I understand dropping the `prov_type` field now, thanks.